Tests/DatadogTests/Datadog/LoggerTests.swift

```diff
@@ -466,55 +483,13 @@ class LoggerTests: XCTestCase {
         logMatchers[2].assertTags(equal: ["env:tests"])
     }
 
-    // MARK: - Sending logs with different network and battery conditions
-
-    func testGivenBadBatteryConditions_itDoesNotTryToSendLogs() throws {
```

Comment:
I'm removing these 2 tests, because we're no longer running full Core backend in Logger tests and the behaviour of conditional uploads is part of the Core. This needs to be tested separately as `FeatureUpload` tests. I plan to add such tests after migrating both Tracing and RUM in similar fashion.